internal/cmd/deploy/deploy.go

```diff
@@ -187,7 +187,7 @@ YAML:
 				}
 
 				if !stackExists {
-					err = cfn.DeleteStack(stackName)
+					err = cfn.DeleteStack(stackName, "")
```

Comment:
Should we also apply the role arn here?

Reply:
Good question. As I understand, we created the stack from scratch and deleting it if the user doesn't deploy the change set. If the user passes the role arn flag, the stack will already have the role set. If no role is passed to the API, it will reuse the correct role. Note that the stack is empty, so CloudFormation might not assume the role as no resources are being touched (no sure about that).
In my view, adding the role ARN flag is a matter of code intelligibility rather than functionality.
What do you think?

Thanks for the reviews!

Reply:
As long as it won't fail, we don't need to add it.